internal/pipe/nfpm/nfpm_test.go

```diff
@@ -1340,6 +1355,76 @@ func TestAPKSpecificScriptsConfig(t *testing.T) {
 	})
 }
 
+// TODO: I need some help on advice to test the IPK specific code properly.  It
```

Comment:
I'll take a look when I have some time

Reply:
@caarlos0 are you waiting for a change to merge this PR in, or just getting time?

Reply:
sorry, it got lost in my notification swamp

merging it now :)